bin/upgrading/update-db-general.pl

```diff
@@ -18,4190 +18,2655 @@
 
 mark_clustered(@LJ::USER_TABLES);
 
-register_tablecreate( "vgift_ids", <<'EOC');
```

Comment:
This file has been cleanly dumped from an updated database, using MySQL 8. This basically compacts all of the alters so they're gone, which means that it should be much easier to reason about what's in the db.